wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ViewModel/ConversationListViewControllerViewModel.swift

```diff
@@ -117,6 +113,18 @@ extension ConversationListViewController.ViewModel {
         }
 
         updateObserverTokensForActiveTeam()
+
+        notificationCenter.addObserver(forName: UIApplication.didBecomeActiveNotification,
+                                       object: nil,
+                                       queue: .main) { [weak self] _ in
+            self?.updateE2EICertifiedStatus()
+        }
+
+        notificationCenter.addObserver(forName: .e2eiCertificateChanged,
+                                       object: nil,
+                                       queue: .main) { [weak self] _ in
+            self?.updateE2EICertifiedStatus()
```

Comment:
@KaterinaWire with that change you need to retain the returned observer token and remove it latest on dealloc:
https://developer.apple.com/documentation/foundation/notificationcenter/1411723-addobserver

 > Return Value
> An opaque object to act as the observer. Notification center strongly holds this return value until you remove the observer registration.

Reply:
Fixed